LICENSE

```diff
@@ -1,29 +1,24 @@
-BSD 3-Clause License
-
-Copyright (c) 2020, Screwdriver
+Copyright Verizon Media
```

Comment:
should update https://github.com/screwdriver-cd/generator-screwdriver/blob/master/app/templates/LICENSE eventually